debian/cloud-init.postrm

```diff
@@ -5,10 +5,6 @@ set -e
 case "$1" in
    purge)
       rm -f /etc/cloud/cloud.cfg.d/90_dpkg.cfg
-      rm -f /etc/apt/apt.conf.d/90cloud-init-pipelining
```

Comment:
Cloud-init's used to have a matching postinst which installed this file (it ran `cloud-init single` from the postinst). Cloud-init no longer automatically installs it, so we can drop this.

Reply:
ack. Dropped in cloud-init 18.5, which means any  package upgrades coming from bionic++ will already have this patch.